### DISCUSSION TITLE
Welcome to esp-box-emu Discussions!

### DISCUSSION THREAD

**Discussion by finger563** [esp-cpp/esp-box-emu#49] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by paultweten** [comment]
Hello! I really love this project. I am curious if anyone has plans to port this over to the m5stack coreS3?

**Reply by finger563** [reply]
To my knowledge no one has, but I can take a look at that board and see if I can whip up a bsp for it. It's similar enough that it should be possible to make it work relatively easily
